modules/oxide/src/less/skins/content/dark/content.less

```diff
@@ -25,12 +25,64 @@ table {
   border-collapse: collapse;
 }
 
-table th,
-table td {
-  border: 1px solid #6d737b;
+/* Apply a default padding if legacy cellpading attribute is missing */
+table:not([cellpadding]) th,
+table:not([cellpadding]) td {
   padding: .4rem;
 }
 
+/* Set default table styles if a table has a positive border attribute
+   and no inline css */
+table[border]:not([border="0"]):not([style*="border-width"]) th,
+table[border]:not([border="0"]):not([style*="border-width"]) td {
+  border-width: 1px;
+}
+
+/* Set default table styles if a table has a positive border attribute
+   and no inline css */
+table[border]:not([border="0"]):not([style*="border-style"]) th,
+table[border]:not([border="0"]):not([style*="border-style"]) td {
+  border-style: solid;
+}
+
+/* Set default table styles if a table has a positive border attribute
+   and no inline css */
+table[border]:not([border="0"]):not([style*="border-color"]) th,
+table[border]:not([border="0"]):not([style*="border-color"]) td {
+  border-color: #6d737b;
+}
+
+/* Cascade down the table border styles to table cells */
+table[border][style*="solid"]:not([border="0"]) th,
```

Comment:
As we're just searching for the keyword appearing somewhere within the inline style, are we worried about false positives with this selector? For example, the word "solid" (or any of the others) appearing in an "outline" or "font-family" declaration.

With the way that the formatter likes to apply styles, this should be a low risk, but I just wanted to make sure we're all aware of the risk (and maybe document it?) moving forward.

Reply:
Can confirm this isn't safe, as these styles could easily come in via copy/paste. http://fiddle.tinymce.com/xthaab is a basic replication case to show having the outline changed the cells to use dashed as well.

@metricjs is going to pull the "safe" stuff out of this PR so that can be included in 5.5.

Reply:
To summarise the slack discussion: we realised this kind of HTML could come in from paste, etc. Word and Excel might even trip this. And from testing we do get false positives.

Reply:
Good catch @ashfordneil! This isn't really solvable. Our dialog currenly assumes integrators start out with a unstyled table and do everything inline, and we know that's not the common use case. Any new table ui will have to be much more configurable to manage already styled tables.